_source/logzio_collections/_log-sources/aws-cross-account.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Ship logs from multiple AWS accounts
+logo:
+  logofile: aws-cloudwatch.svg
+  orientation: vertical
+data-source: Multiple AWS accounts simultaneously
+data-for-product-source: Logs
+short-description: Send logs from multiple AWS accounts to Logz.io.
+templates: ["lambda-cloudwatch", "cloudformation"]
+open-source:
+  - title: AWS cross account shipping
+    github-repo: aws-cross-account
+contributors:
+  - nshishkin
+shipping-tags:
+  - aws
+order: 110
+---
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Overview](#overview)
+* [Deploy integration](#deploy)
+* [Update deployed integration](#update)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="overview">
+
+Deploy this integration to simultaneously ship logs from multiple AWS accounts to Logz.io.
+
+The integration consists of the following main steps:
+
+* Creating a **landing account**, which receives logs from your multiple AWS accounts and sends them to Logz.io.
+* Creating **destinations in the landing account** on each region you need to send logs from.
+* Creating **subscription filters** in your multiple AWS accounts to enable them to send logs from Cloudwatch to the **destinations in the landing account**.
+
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="deploy">
+
+
+**Before you begin, you'll need**:
+
+* Your AWS service [publishes logs to Cloudwatch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/aws-services-sending-logs.html). 
+* Your log group is in the format: `/<<AWS-PARTITION/<<SERVICE-NAME>>/<<LOG-GROUP-NAME>>`, for example: `/aws/lambda/my_function`.
+* AWS CLI
+
+<div class="tasklist">
+
```

Comment:
Can we add a sub-header to make it clear that steps 1-10 are in the landing account?